concent_api/api_testing_helpers.py

```diff
@@ -73,7 +74,10 @@ def api_request(host, endpoint, private_key, public_key, data = None, headers =
         )
         print('STATUS: {} {}'.format(response.status_code, http.client.responses[response.status_code]))
         print('MESSAGE:')
-        print_golem_message(decoded_response, private_key, public_key)
+        if response.headers['Content-Type'] != 'application/octet-stream':
+            print(json.loads(response.content.decode())['error'])
```

Comment:
This is still wrong. If the content type is HTML, it will try to decode it as if it were JSON.

Reply:
Change statement to check if this is application/json